src/quatrex/core/quatrex_config.py

```diff
@@ -150,6 +150,12 @@ class CoulombScreeningConfig(BaseModel):
     solver: Literal["rgf", "inv"] = "rgf"
     obc: OBCConfig = OBCConfig()
     lyapunov: LyapunovConfig = LyapunovConfig()
+    eta: NonNegativeFloat = 1e-6  # eV
+
+    temperature: PositiveFloat = 300.0  # K
+
+    left_temperature: PositiveFloat | None = None
+    right_temperature: PositiveFloat | None = None
```

Comment:
I see that you compute the occupancies from these temperatures in the `CoulombScreeningSolver` but never use them. Do you think we should still keep these temperatures in the config?

Reply:
That was mainly for testing the fluctuation-dissipation theorem of W, but it did seem to work. I would like to do some more testing before we can discard them